frontend/io-webapp/src/session/Student.js

```diff
@@ -1,11 +1,41 @@
-import React from 'react'
+import React from "react";
 
-const Student = () => {
-    return (
-        <div>
-            <h1>Here will be student view</h1>
-        </div>
-    )
-}
+import Chat from "./Chat";
+import Participants from "./Participants"
+import Question from "./Question";
 
-export default Student
+import "./Session.css";
+import Sessionbar from "./Sessionbar";
+
+const Student = (props) => {
+  const [chat, setChat] = React.useState(false);
+  const [participants, setParticipants] = React.useState(false);
```

Comment:
To może tylko osobista preferencja, ale lubię jeśli nazwa zmiennej wskazuje na jej typ. W szczególności tutaj `participants` jak dla mnie jasno sugeruje, że ta zmienna będzie przechowywać jakąś kolekcję (l. mnoga). Tak samo dla booli lubię konwencję "isPies" :D

Reply:
Masz rację, w teorii miałem zamiar dodawać is przed boolami, z tego co kojarzę tak gdzieś też zrobiłem, ale miejscami o tym zapominam.